assignments/arrays.js

```diff
@@ -63,34 +63,63 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+for (let i = 0; i < inventory.length; i++) {
+    let car = inventory[i];
+    for (let property in car) {
+        if (car[property] === 33) {
+            console.log('Car 33 is a', car.car_year, car.car_make, car.car_model);
+        }
```

Comment:
```suggestion
        console.log(`Car 33 is a ${car33.car_make} ${car33.car_model} ${car33.car_year}` );
```

using template literals.